env_spec.py

```diff
@@ -18,12 +18,15 @@
 
 def create_list(input_str):
     """
-    Creates a list from input_str_list.Returns the new list.
+    Creates a list from input_str_list.Returns the new list. If input_str=[1,2,], returns empty string(syntax error).
     """
     input_str = input_str.replace("[", "")
     input_str = input_str.replace("]", "")
     new_list = input_str.split(",")
 
+    if "" in new_list:
+        return ""
```

Comment:
This is expected to return a list, but here you return an empty string. I think it would be better if you returned `[]`.